modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,48 @@
+= Optimize Compaction Settings
```

Comment:
nit: i'd remove "Optimize" and just leave this "Compaction Settings".

Reply:
+1